src/Components/CollectionListItem/ListItem.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+function ListItem({ text, number, color }) {
```

Comment:
please name this well
```suggestion
function CollectionListItem({ text, number, color }) {
```